mlir/patches/mhlo-Add-PassesIncGen-in-transforms-CMakeList.patch

```diff
@@ -0,0 +1,24 @@
+From 431da821402484bfa128d0559788c86e33f14dc2 Mon Sep 17 00:00:00 2001
+From: Tzung-Han Juang <tzunghan.juang@gmail.com>
+Date: Fri, 26 Jul 2024 10:39:45 -0400
+Subject: [PATCH 1/1] Add PassesIncGen in transforms CMakeList
+
+---
+ mhlo/transforms/CMakeLists.txt | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/mhlo/transforms/CMakeLists.txt b/mhlo/transforms/CMakeLists.txt
+index ecec1370..9e80604f 100644
+--- a/mhlo/transforms/CMakeLists.txt
++++ b/mhlo/transforms/CMakeLists.txt
+@@ -195,6 +195,7 @@ add_mlir_library(ChloPasses
+   MLIRhlo_opsIncGen
+   MLIRChloLegalizeToHloIncGen
+   MLIRMhloPassIncGen
++  PassesIncGen
```

Comment:
This again ? 😅  Why do they keep messing up their build scripts :/ Do we have a PR for this upstream?

Reply:
Not yet. I will open one later today.

Reply:
They keep messing their build scripts because they don't use single core to build them. These errors don't show up if you are building sequentially. They probably have dedicated servers with caching and what not, so it makes it more difficult for them to see these errors.

Reply:
Upstream commit: https://github.com/openxla/xla/pull/15446#issuecomment-2256331627